aws/resource_aws_codebuild_project.go

```diff
@@ -324,6 +358,29 @@ func expandProjectEnvironment(d *schema.ResourceData) *codebuild.ProjectEnvironm
 	return projectEnv
 }
 
+func expandVpcConfig(d *schema.ResourceData) *codebuild.VpcConfig {
```

Comment:
We really try to limit the usage of `*schema.ResourceData` in downstream functions unless its necessary in multiple pieces of logic. It helps make the code more testable and reusable (if necessary in the future). Could you update this function's signature to `expandCodeBuildVpcConfig(m map[string]interface{}) *codebuild.VpcConfig`? Thanks!